.vue.webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const webpack = require('webpack');
+const path = require('path');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const VueLoaderPlugin = require('vue-loader').VueLoaderPlugin;
+
+const config = {
+    entry: {
+        'vue': './lib/bundles/vue.js',
+    },
+    output: {
+        path: path.resolve(__dirname, 'public/lib/vue'),
+        publicPath: '/public/lib/vue/',
+        asyncChunks: true,
+        clean: true,
+    },
+    module: {
+        rules: [
+            {
+                // Vue SFC
+                test: /\.vue$/,
+                loader: 'vue-loader'
+            },
+            {
+                // Build styles
+                test: /\.css$/,
+                use: [MiniCssExtractPlugin.loader, 'css-loader'],
+            },
+        ]
+    },
+    plugins: [
+        new VueLoaderPlugin(), // Vue SFC support
+        new MiniCssExtractPlugin(),
+        new webpack.ProvidePlugin(
+            {
+                process: 'process/browser', // required by some libs (including `popper.js`)
+            }
+        ),
+        new webpack.DefinePlugin({
+            __VUE_OPTIONS_API__: false, // We will only use composition API
+            __VUE_PROD_DEVTOOLS__: false,
+        }),
+    ],
+    resolve: {
+        fallback: {
+            'process/browser': require.resolve('process/browser.js'), // required by @lezer/lr (codemirror dependency)
+            "path": require.resolve("path-browserify"),
+        },
+    },
+    watchOptions: {
+        ignored: [
+            path.resolve(__dirname, 'lib/bundles'),
+            path.resolve(__dirname, 'node_modules'),
+            path.resolve(__dirname, 'vendor'),
+            path.resolve(__dirname, 'public'),
+            // ignore js except for js/src
+            path.resolve(__dirname, 'js/!(*src)'),
+        ]
+    },
```

Comment:
As we use a dedicated config now, not sure it is mandatory. As far as I understand, `webpack` will watch only resolved files, so it probably means that it will watch only the `lib/bundles/vue.js` and  the `js/src` files.